lib/features/email/domain/state/get_detailed_email_by_id_state.dart

```diff
@@ -9,15 +9,14 @@ class GetDetailedEmailByIdLoading extends UIState {}
 
 class GetDetailedEmailByIdSuccess extends UIState {
 
-  final Email email;
-  final DetailedEmail detailedEmail;
+  final Map<Email, DetailedEmail> mapDetailedEmail;
   final AccountId accountId;
   final Session session;
 
-  GetDetailedEmailByIdSuccess(this.email, this.detailedEmail, this.accountId, this.session);
+  GetDetailedEmailByIdSuccess(this.mapDetailedEmail, this.accountId, this.session);
```

Comment:
why don't you use required here, because 3 parameters is pretty long

Reply:
We should only use `required` when the parameters are ambiguous and it is required. IMO, it's not so important while variables are already too obvious for developers.